tests/providers/google/cloud/hooks/test_dataprep.py

```diff
@@ -72,7 +85,7 @@ def test_should_not_retry_after_success(self, mock_get_request):
         ],
     )
     def test_should_retry_after_four_errors(self, mock_get_request):
-        self.hook.get_jobs_for_job_group.retry.sleep = mock.Mock()  # pylint: disable=no-member
+        self.hook.get_jobs_for_job_group.retry.sleep = (mock.Mock())  # pylint: disable=no-member
```

Comment:
This is strange, why?

Reply:
Black formatting, I will remove redundant parenthesis.